packages/@divvi/mobile/src/analytics/Events.tsx

```diff
@@ -625,9 +594,6 @@ export enum JumpstartEvents {
   jumpstart_reclaim_contact_support = 'jumpstart_reclaim_contact_support',
   jumpstart_claim_status_fetch_success = 'jumpstart_claim_status_fetch_success',
```

Comment:
i think the claim_status_* events can also be removed, since they're for when a user claims the funds via the dynamic link